cypress/e2e/home_spec.cy.js

```diff
@@ -0,0 +1,38 @@
+// TEST: HOME PAGE //
+
+/* eslint-disable no-undef */
+describe('home page', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+    cy.intercept('POST', 'https://everuse-be-b6017dbfcc94.herokuapp.com/graphql', {
+      statusCode: 201,
+      fixture: 'items.json'
+    }).as('getItems')
+    cy.wait('@getItems')
```

Comment:
Nice work waiting for this in beforeEach

Reply:
Thank you